doc/Makefile

```
@@ -1,10 +1,12 @@
 # Makefile for Sphinx documentation
 
-SPHINXOPTS    = -a -n -W
+#SPHINXOPTS    = -a -n -W
+SPHINXOPTS    = -a -n
```

```suggestion
SPHINXOPTS    = -a -n -W
```
@MichaelLueken @gsketefian Removing the `-W` option permanently will prevent the docs from catching problems in the docs. It needs to be added back for the PR to be merged, although it's completely acceptable (and recommended!) for troubleshooting.

---

@gspetro-NOAA Yep, I know.  I was intending to change all that back.  Let me try `autodoc_mock_imports` setting and once things are working, I'll revert `Makefile` to its original form.

---

@gspetro-NOAA `Makefile` should now be back to its original form before this PR.

---

The changes to `SPHINXOPTS` have been reverted in `doc/Makefile`.

Resolving conversation now.